HW_3/Task2.py

```diff
@@ -0,0 +1,11 @@
+#Дан список повторяющихся элементов. Вернуть список с дублирующимися элементами. 
+#В результирующем списке не должно быть дубликатов.
+
+list = [1, 1, 2, 2, 2, 3, 3, 3, 4, 6, 6, 7]
+
+result = set()
+for i in list:
+    spam = list.count(i)
+    if spam > 1:
+        result.add(i)
```

Comment:
Вы всегда получаете квадратичную сложность. Но ее можно немного понизить.
```suggestion
for i in set(list):
    spam = list.count(i)
    if spam > 1:
        result.add(i)
```
И больше нигде множество использовать не обязательно